code/__HELPERS/game.dm

```diff
@@ -328,6 +328,8 @@
 
 /proc/considered_alive(datum/mind/M, enforce_human = TRUE)
 	if(M && M.current)
+		if(M.has_antag_datum(/datum/antagonist/changeling))

```

Comment:
This would bypass the enforce_human check though, not sure if that's a good idea or not.

Reply:
can you put this in ~337 and make that return statement not dumb? you can have a bunch of if's return false instead of a really convoluted and dumb single return.